src/SnoopCompile.jl

```diff
@@ -25,29 +27,32 @@ function snoop(filename, commands)
     # addprocs will run the unmodified version of julia, so we
     # launch it as a command.
     code_object = """
-        while !eof(STDIN)
-            eval(Main, deserialize(STDIN))
-        end
-        """
-    in, io = open(`$(Base.julia_cmd()) --eval $code_object`, "w", STDOUT)
-    serialize(in, quote
-        import SnoopCompile
+            using Serialization
+            using Pkg
+            Pkg.activate(abspath("."))
```

Comment:
If this is necessary can't we just tell people to do this before running `@snoop`?

Reply:
That won't work, since it must be included in the new process that is being spawned.  I am actually a little confused as to the intended difference in purpose between `@snoop` and `@snoop1`?